### ISSUE TITLE
formatOnPaste and formatOnSave have different results

### ISSUE THREAD

**Issue by donovanhiland** [prettier/prettier-vscode#343]
Recently setup vscode and prettier on a new machine and started noticing that the results of formatting on paste and formatting on save are different.

**Format on Paste**
```javascript
<SomeComponent longProperty1="longPropertyValue" longProperty2="longPropertyValue2" longProperty3="longPropertyValue3" />
```

**Format on Save**
```javascript
<SomeComponent
  longProperty1="longPropertyValue"
  longProperty2="longPropertyValue2"
  longProperty3="longPropertyValue3"
/>
```

**.prettierrc**
```json
{
  "semi": false,
  "singleQuote": true,
  "trailingComma": "all",
  "overrides": [
    {
      "files": ".prettierrc",
      "options": {
        "parser": "json"
      }
    }
  ]
}
```

Any suggestions on what might be happening or where I can look to track down the problem?

**Comment by CiGit** [comment]
Did you paste in an empty document?

Format on paste uses the range formatter vs the document formatter for format on save

Maybe an issue with the range formatter.

**Comment by ntotten** [comment]
This is fixed in version 3.0. Please reopen if you still see the problem. 

You can install the preview of version 3.0 here: https://marketplace.visualstudio.com/items?itemName=Prettier.prettier-vscode
